Nodejs/Product/Nodejs/Snippets/1033/Nodejs/readstream.snippet

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>readstream</Title>
+      <Description>Code Snippet for streaming data</Description>
+      <Shortcut>readstream</Shortcut>
+      <Author>Microsoft Corporation</Author>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>resource</ID>
+          <Default>resource</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[var stream = fs.createReadStream('$resource$');

```

Comment:
use ```var readStream = ``` rather than ```var stream =```, as it's more descriptive. Additionally, use $path$ rather than $resource$ as that is actually how the node docs describe the argument to ```createReadStream```